src/components/AppNavigation/CalendarList/PublicCalendarListItem.vue

```diff
@@ -6,12 +6,14 @@
 <template>
 	<AppNavigationItem :loading="calendar.loading"
 		:name="calendar.displayName || $t('calendar', 'Untitled calendar')"
-		:menu-open.sync="menuOpen"
-		@click.prevent.stop="toggleEnabled">
+		:menu-open.sync="menuOpen">
 		<template #icon>
 			<AppNavigationIconBullet v-if="calendar.enabled"
 				:color="calendar.color"
 				@click.prevent.stop="toggleEnabled" />
+			<AppNavigationIconBullet v-else
+				:color="calendar.color"
+				@click.prevent.stop="toggleEnabled" />
```

Comment:
```suggestion
		:menu-open.sync="menuOpen">
		<template #icon>
			<AppNavigationIconBullet :color="calendar.color" />
```

This is actually a bug. Toggling should not be available in public views because the enabled state cannot be persisted anyway.